canonicalwebteam/blog/wordpress_api.py

```diff
@@ -43,16 +43,16 @@ def get_articles(tags, per_page=12, page=1, exclude=None, category=None):
     return process_response(response), total_pages
 
 
-def get_article(tags, slug):
-    url = "".join(
-        [
-            API_URL,
-            "/posts?slug=",
-            slug,
-            "&tags=",
-            "".join(str(tag) for tag in tags),
-        ]
-    )
+def get_article(slug, tags=None, excluded_tags=None):
```

Comment:
Actually the API will happily accept empty values for both `tags` and `exclude_tags` - https://admin.insights.ubuntu.com/wp-json/wp/v2/posts?tag=&exclude_tags= - which opens up the opportunity for the code to be significantly simpler.

Also, if the argument to the API is called "exclude_tags", that also seems descriptive enough for our use here. To use a very similar but slightly different variable name "excluded_tags" seems likely to cause confusion.

What do you think of:

``` python3
def get_article(slug, tags=[], exclude_tags=[]):
    url = (
        f"{API_URL}/posts?slug={slug}&tags={','.join(tags)}"
        f"&exclude_tags={','.join(exclude_tags)}"
    )
```

Reply:
Have updated. Whaddaya think?